src/stepfunctions/steps/integration_resources.py

```diff
@@ -25,6 +29,17 @@ class IntegrationPattern(Enum):
     WaitForTaskToken = "waitForTaskToken"
     WaitForCompletion = "sync"
     RequestResponse = ""
+    WaitForCompletionWithJsonResponse = "sync:2"
+
+
+class ServiceIntegrationType(Enum):
+    """
+    Service Integration Types for service integration resources (see `Service Integration Patterns <https://docs.aws.amazon.com/step-functions/latest/dg/connect-to-resource.html`_ for more details.)
+    """
+
+    REQUEST_RESPONSE = "RequestResponse",
+    RUN_A_JOB = "RunAJob",
```

Comment:
this is just called `RUN_JOB` in the CDK. I would think that we should try and stick to something similar?